scripts/run-treecompose

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/bash
+set -xeuo pipefail
+
+repo=$1; shift
+ref=$1; shift
+
+# We don't keep history in the tree.
+previous_commit=$(ostree --repo="${repo}" rev-parse "${ref}" || true)
+rm ${repo}/refs/heads/* -rf
+# This should be the default.  It'll help us trace down a
+# coredump we're seeing periodically.
+export G_DEBUG=fatal-warnings
+rpm-ostree compose tree --repo=${repo} "$@"
+commit=$(ostree --repo=${repo} rev-parse ${ref})
+version=$(ostree --repo=${repo} show --print-metadata-key=version ${commit} | sed -e "s,',,g")
```

Comment:
This works for us, since we add the metadata `version` string when the script is called, but if someone were to reuse this script, this could bomb out.

Probably not a deal-breaker, but thought I would mention it.

Reply:
True.  My thoughts on reusability are that this commit is stage 2 - "stage 3" of reusability would be having it in coreos-assembler or so, and at that point we'd hopefully be using a not-bash language and be more flexible.